App/components/Input/input.styled.ts

```diff
@@ -0,0 +1,39 @@
+import { TextInputProps } from 'react-native'
+import styled from 'styled-components/native'
+import theme from '../../theme/index'
+
+type Props = {
+  size: keyof typeof theme.inputSizes
+  editable: boolean
+}
+
+export const StyledInput = styled.TextInput<Props & TextInputProps>`
+  margin-left: 5px;
+  font-size: ${({ size, theme }) => theme.fontSizes[size]};
+  overflow: hidden;
+  width: 315px;
+`
+
+export const StyledView = styled.View<Props>`
+  display: flex;
+  flex-direction: row;
+  border: 1px solid #9e9e9e;
```

Comment:
please consider adding a new color to the theme

Reply:
done